index.html

```diff
@@ -2833,136 +2833,105 @@ <h3>Consumption</h3>
-                <pre class="example">
+        <pre class="example"
+          title="A valid serialization of a simple @context entry">
 {
   "@context": "https://www.w3.org/ns/did/v1",
   ...
 }
-                </pre>
-              </li>
-              <li>
-A <a href="https://tools.ietf.org/html/rfc8259#section-5">JSON Array</a>,
-with first item the <a href="https://tools.ietf.org/html/rfc8259#section-7">JSON
-String</a> <code>https://www.w3.org/ns/did/v1</code>, and subsequent items of
-type <a href="https://tools.ietf.org/html/rfc8259#section-7">JSON String</a> or
-<a href="https://tools.ietf.org/html/rfc8259#section-4">JSON Object</a>.
-                <pre class="example">
+        </pre>
+        <pre class="example"
+          title="A valid serialization of a layered @context entry">
 {
   "@context": [
     "https://www.w3.org/ns/did/v1",
-    "https://example.com/blockchain-identity/v1"
+    "https://did-method-extension.example/v1"
   ],
   ...
 }
-                </pre>
-              </li>
-            </ul>
+        </pre>
 
-            <p>
-All JSON-LD terms defined by the <code>@context</code> SHOULD exist in the DID
-Specification Registries [[?DID-SPEC-REGISTRIES]] in order to achieve
-interoperability across different <a>representations</a>. If a term does not exist
-in the DID Specification Registries, then the <a>DID document</a> might not be
-interoperable across <a>representations</a>.
-            </p>
+        <p class="advisement" title="JSON-LD serialization rules">
+All implementers creating <a>conforming producers</a> that produce JSON-LD
+<a>representations</a> are advised to ensure that their algorithms
+produce valid JSON-LD [[JSON-LD11]] documents. Invalid JSON-LD documents will
+cause JSON-LD processors to halt and report errors.
+        </p>
+
+        <p>
+In order to achieve interoperability across different <a>representations</a>,
+all JSON-LD Contexts and their terms SHOULD be registered in the DID
+Specification Registries [[?DID-SPEC-REGISTRIES]].
+        </p>
 
-            <p>
-It is RECOMMENDED that dereferencing each <a>URI</a> value of the
-<code>@context</code> entry results in a document containing
-machine-readable information about the context. Note that further expectations
-of additional JSON-LD contexts are described as part of the DID Specification
-Registries registration process.
-            </p>
-          </dd>
-        </dl>
         <p>
-Producers SHOULD NOT produce <a>DID documents</a> that contain properties not
-defined via the <code>@context</code>. Properties that are not defined via the
-<code>@context</code> MAY be dropped by Consumers.
+A <a>conforming producer</a> that generates a JSON-LD <a>representation</a>
+SHOULD NOT produce a <a>DID document</a> that contains terms not defined
+via the <code>@context</code> as <a>conforming consumers</a> might remove
+unknown terms. When serializing a JSON-LD <a>representation</a> of a <a>DID
```

Comment:
```suggestion
via the <code>@context</code> as <a>conforming consumers</a> are expected
to remove unknown terms. When serializing a JSON-LD <a>representation</a> of a <a>DID
```

In Consumption it says dropping undefined terms is a SHOULD, so suggest strengthening the language here a bit to match.